### PR TITLE
Update install_ruby_on_rails.md

### PR DESCRIPTION
**Pull request by TeriyakiBomb**
Added references to alternative version management for Ruby installs. Current guide mentions others, but focuses only on installing with Mise. Might infer that it's the only way.

Not sure whether or not to include RVM.

### Motivation / Background

This Pull Request has been created because new guide references a single ruby package manager.

### Detail

This Pull Request changes install_ruby_on_rails.md

### Checklist

Before submitting the PR make sure the following are checked:

* [x] This Pull Request is related to one change. Unrelated changes should be opened in separate PRs.
* [x] Commit message has a detailed description of what changed and why. If this PR fixes a related issue include it in the commit message. Ex: `[Fix #issue-number]`
* [x] Tests are added or updated if you fix a bug or add a feature.
* [x] CHANGELOG files are updated for the changed libraries if there is a behavior change or additional feature. Minor bug fixes and documentation changes should not be included.

### CODE REVIEW

File: guides/source/install_ruby_on_rails.md [hunk 1]
```diff
@@ -7,8 +7,16 @@ This guide will walk you through installing the Ruby programming language and th
 
 While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
 
+You can also use:
+- [ASDF](https://asdf-vm.com)
+- [rbenv](https://rbenv.org)
+- [chruby](https://github.com/postmodern/chruby)
+
```
Comment (Venkat-RK) [inline guides/source/install_ruby_on_rails.md:14]:
```suggestion
Please refer https://www.ruby-lang.org/en/documentation/installation/#managers to explore alternate options.
```

File: guides/source/install_ruby_on_rails.md [hunk 2]
```diff
@@ -7,8 +7,16 @@ This guide will walk you through installing the Ruby programming language and th
...
+You can also use:
+- [ASDF](https://asdf-vm.com)
+- [rbenv](https://rbenv.org)
+- [chruby](https://github.com/postmodern/chruby)
+Please refer https://www.ruby-lang.org/en/documentation/installation/#managers to explore alternate options.
 Alternatively, you can use Dev Containers to run Rails without installing Ruby or Rails directly on your machine. Check out the [Getting Started with Dev Containers](getting_started_with_devcontainer.html) guide to learn more.
 
+For this guide,  we'll demonstrate installing Ruby with Mise.
+
+
```
Comment (akhilgkrishnan) [inline guides/source/install_ruby_on_rails.md:19]:
```suggestion
```

File: guides/source/install_ruby_on_rails.md [hunk 3]
```diff
@@ -7,8 +7,16 @@ This guide will walk you through installing the Ruby programming language and th
 
 While your OS might come with Ruby pre-installed, it's often outdated and can't be upgraded. Using a version manager like [Mise](https://mise.jdx.dev/getting-started.html) allows you to install the latest Ruby version, use a different Ruby version for each app, and easily upgrade to new versions when they're released.
 
+You can also use:
+- [ASDF](https://asdf-vm.com)
+- [rbenv](https://rbenv.org)
+- [chruby](https://github.com/postmodern/chruby)
```
Comment (akhilgkrishnan) [inline guides/source/install_ruby_on_rails.md:13]: Seems `chruby` is not maintaining properly.

Comment (TeriyakiBomb) [reply guides/source/install_ruby_on_rails.md:13]: I’m not well versed enough in the specific state of chruby to know for sure

**Comment by zzak** [comment]
I think the point of this guide is to give a single, easy to follow path for setting up Ruby. We don't need to complicate it by suggesting many additional tools, so while I personally use `rbenv` everywhere can see the benefit of keeping this guide super simple.

**Comment by TeriyakiBomb** [comment]
I don’t really see the issue with “we’re installing it this way in this guide, but others exist.” But oh well.

**Pull request closed without merging**